### ISSUE TITLE
‘Withdraw unavailable due to risk management’

### ISSUE THREAD

**Issue by realpheli** [binance-us/binance-official-api-docs#27]
Oh yeah, Binance.US is quick to take their cut of your finances from their 'minimum' deposit/withdraw/transfer requirements. Yet can't even move my assets out of here to an actual function exchange. Doesn't matter if I try to withdraw crypto, or transfer, or even try to cash out and go ach deposit, same error message every time. And the only word so far is, 'we are overwhelmed with support tickets, we will merge all of your request into one, PLEASE DO NOT CONTACT US/ create any more tickets for this issue. We will get to you when we find out how to run customer support' pathetic.

**Comment by captaincobb92** [comment]
Did they ever allow you to withdrawal your funds or respond to you with anything other than, "stop sending ticket submissions"?

I have been dealing with the exact same issue and it's been well over a month and I haven't been helped or informed of when help is coming.  I have a lawyer going over the user agreement now to see if there are any means of making binance.us to actually address this issue and at least admit that they have committed felony grand larceny for accounts holding over 1k in assests.  This is retarded to have to go to these lengths just to have access to our own finances.  If they would have addressed this issue within 10 to 14 days, then I would have been happy and continued to support the company, but I have no intention of ever engaging or using the services provided by binance.us again, once I have collected all of my assests they have been what is obviously "intentional theft" waiting for all their victims to give up and forget about their stolen property.

**Comment by realpheli** [comment]
Nope. The last messsge I got from them was ‘we noticed a solution hasn’t been found for your issue, reply to this email if you still need help’. I’m like you clearly see that it’s unresolved yet i have to respond to this message in order for you guys to continue. Like wtf right? I tried everything. Tried clearing cache, tried to just transfer different coins to another wallet, tried deleting bank info and resubmit it. Nothing. Then I tried tweeting them. Nothing. Even sent more messages thru email chain, nothing. 

R/S
Le Phi H

**Comment by frankdes62** [comment]
Im having same issues - did anyone find an answer?

**Comment by realpheli** [comment]
I think i finally found a possible solution for 'suspended due to risk management' 

Apparently it says to fix it, you have to make a deposit. Bull f'n Shizz. The whole reason why I'm in this mess is because my initial deposit with my debit card for only 20 bucks gave an error and 'deposit failed' message. So I did it again, same thing. So instead I just did an ACH with bank, deposit failed, tried again and finally worked. But then when I got back to the wallet page, the two debit was credited in my account. I was charged BOTH deposits with debit card. Even though it says failed on trans history. So went back and canceled ach. Got the cancel. BUT WAS CHARGED few days later when. F Binance.uS. 

https://support.binance.us/hc/en-us/articles/360047428953-Why-am-I-Not-Able-to-Withdraw-or-Trade-



![image](https://user-images.githubusercontent.com/83050699/120379972-88ea6600-c2ee-11eb-9a5e-875f1fa592a4.png)
